external-deps/spyder-remote-services/spyder_remote_services/services/files/base.py

```
@@ -433,3 +443,53 @@ def fs_copy(self, src_str: str, dst_str: str, metadata: bool=False):
+                        mode=0x7777 & p.stat().st_mode,
+                        size=p.stat().st_size,
+                        method=CompressionType.ZIP_64,
+                    )
+                )
+
+        try:
+            if not zip_files:
+                yield None
+            else:
+                yield ZipStream(zip_files,
+                    get_compressobj=lambda: zlib.compressobj(
```

Flake8 (7.2.0)

[error] continuation line under-indented for visual indent

(E128)